src/main/resources/commandblockide.accesswidener

```diff
@@ -1,2 +1,4 @@
 accessWidener	v1	named
 accessible	class	net/minecraft/client/gui/EditBox$Substring
+
+accessible method net/minecraft/block/entity/CommandBlockBlockEntity readNbt (Lnet/minecraft/nbt/NbtCompound;Lnet/minecraft/registry/RegistryWrapper$WrapperLookup;)V
```

Comment:
It would be better to use a Mixin `@Invoker` for this instead of an access widener. The only reason an access widener is used for the `Substring` class is because Mixin accessors don't work for classes.

Reply:
I thought using the access widener was a good thing because it makes it easier to access, but using @Invoker is a better way.
Thanks for sharing this information.